templates/schemavalidator.gotmpl

```diff
@@ -0,0 +1,1194 @@
+{{ define "primitivefieldcontextvalidator" }}
```

Comment:
This seems hard to review / reason about. Any tips on where good documentation is? How did you arrive at this?

(obviously it's "fine" because the proof that this works is in the generated files themselves: if they look okay, then we're okay!)

Reply:
extremely! I started here: https://goswagger.io/generate/cli.html#cli-usage. I cloned the swagger repo locally and grep'd and tweaked into I eventually found the lines that I needed to change. It's not too difficult to diff the swagger repo file with ours, but I admit the maintenance story here isn't fun

Reply:
All good! I think dropping a note about this in the README would be nice so people know the origins of this / where to look when we want to update it.